easybuild/easyconfigs/j/Java/Java-1.8_191-b26-OpenJDK.eb

```diff
@@ -0,0 +1,64 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+name = 'Java'
+easyblock = 'ConfigureMake'
+version = '1.8_191-b26-OpenJDK'
+
+homepage = "https://openjdk.java.net/"
+description = """An open-source implementation of the Java Platform, Standard Edition"""
+
+toolchain = {'name': 'GCCcore', 'version': 'system'}
```

Comment:
@edmondac This should be `toolchain = SYSTEM`?

Cfr. https://easybuild.readthedocs.io/en/latest/System_toolchain.html

Reply:
Ahah!